weatheril/__init__.py

```diff
@@ -89,7 +89,7 @@ def get_current_analysis(self):
                                u_v_index=int(analysis_data.get("u_v_index", "0") or "0"),
                                u_v_level=analysis_data.get("u_v_level"),
                                u_v_i_max=int(analysis_data.get("u_v_i_max", "0") or "0"),
-                               u_v_i_factor=int(analysis_data.get("u_v_i_factor", "0") or "0"),
+                               u_v_i_factor=int(float(analysis_data.get("u_v_i_factor", "0")) or "0"),
```

Comment:
```suggestion
                               u_v_i_factor=float(analysis_data.get("u_v_i_factor", "0") or "0"),
```

Reply:
Also, need to modify `u_v_i_factor` type
https://github.com/t0mer/py-weatheril/blob/main/weatheril/weather.py#L24